contracts/core/RebalancingSet.sol

```diff
@@ -0,0 +1,389 @@
+/*
```

Comment:
Maybe we can move the SetToken, SetTokenFactory, RebalancingSet, and RebalancingSetFactory into their own directory

Reply:
Sure...I'll leave the organization to @asoong